egs2/wsj0_2mix/enh1/conf/tuning/train_enh_tasnet.yaml

```diff
@@ -0,0 +1,43 @@
+optim: adam
+init: xavier_uniform
+max_epoch: 100
+batch_type: folded
+batch_size: 16
+iterator_type: chunk
+chunk_length: 32000
+num_workers: 4
+optim_conf:
+    lr: 1.0e-03
+    eps: 1.0e-08
+    weight_decay: 0
+patience: 4
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - si_snr
+    - max
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 1
+scheduler: reducelronplateau
+scheduler_conf:
+    mode: min
+    factor: 0.5
+    patience: 1
+enh: tasnet
+enh_conf:
+    fs: 8000
+    N: 256
```

Comment:
I think it's better to follow the name used in uPIT (or other front-end configs)